src/Pdf/utils.js

```diff
@@ -1,31 +1,45 @@
 import { jsPDF } from 'jspdf'
 import ugh from 'ugh'
 
-export function convertSvgToTemplate (svg, inputs) {
-  const svgDoc = svg.contentDocument
+import OL_KIT_MARK from 'images/ol_kit_mark.svg'
+
+export function convertSvgToTemplate (svgString, inputs) {
+  const parser = new DOMParser()
+  const svgDoc = parser.parseFromString(svgString, 'application/xml')
+  const svgContainer = svgDoc.children[0]
+  const svgHeight = svgContainer.getAttribute('height')
+  const svgWidth = svgContainer.getAttribute('width')
+
+  // calc unit from svg
+  let unit
+  if (svgHeight.split('mm').length > 1) unit = 'mm'
+  else if (svgHeight.split('cm').length > 1) unit = 'cm'
+  else if (svgHeight.split('m').length > 1) unit = 'm'
+  else if (svgHeight.split('px').length > 1) unit = 'px'
+  else if (svgHeight.split('pt').length > 1) unit = 'pt'
+  else if (svgHeight.split('in').length > 1) unit = 'in'
+  const numberify = string => {
+    const [numberAsString] = string.split(unit) // closure, order matters
+
+    return Number(numberAsString)
+  }
+  // calc dimensions from svg
+  const height = numberify(svgHeight)
+  const width = numberify(svgWidth)
+  const orientation = width > height ? 'landscape' : 'portrait'
```

Comment:
What if (not likely), the width and height are in different units? Are we going to break this orientation computation?

Reply:
that would be pretty edgy but yea i could see a `110mm` height with a `100%` width that may break this?

Reply:
Agreed... just worried that it wouldn't be obvious why it was behaving oddly. Perhaps merely asserting that the units are the same it good enough?